check-symbiosis.lic

```diff
@@ -0,0 +1,80 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#check-symbiosis
+=end
+
+custom_require.call(%w[common common-arcana drinfomon spellmonitor])
+
+class SymbiosisCheck
+
+  def initialize
+    arg_definitions = [
+      [
+        {
+          name: 'symbiosis_type',
+          optional: true,
+          options: %w[activate avoid cast discern endure examine explore harness harvest heal impress learn perform remember resolve spell spring strengthen watch],
+          description: "Which symbiosis do you want to check? Argument -> Yaml -> defaults to harness"
+        }
+      ]
+    ]
+    args = DRC.parse_args(arg_definitions)
+
+    symbiosis_type = args.symbiosis_type || get_settings.symbiotic_research_type || 'harness'

```

Comment:
Just remembered, please add a new entry to base.yaml for `symbiotic_research_type: harness`

This ensures everyone gets the default and then here in code you can remove the `|| 'harness'` because the default is now defined in settings.

Reply:
can do

Reply:
removed harness default from check-symbiosis.lic
added new base.yaml entry